### ISSUE TITLE
"backtrace" listed as a feature, but can not be compiled without it

### ISSUE THREAD

**Issue by sstelfox** [purpliminal/rust-dotenv#107]
When attempting to compile this crate with "default-features = false" the build fails to compile with the following error message:

```
error[E0046]: not all trait items implemented, missing: `extract_backtrace`
 --> /home/sstelfox/.cargo/registry/src/github.com-1ecc6299db9ec823/dotenv-0.12.0/src/errors.rs:3:17
  |
3 | #[derive(Debug, ErrorChain)]
  |                 ^^^^^^^^^^ missing `extract_backtrace` in implementation
  |
  = note: `extract_backtrace` from trait: `fn(&std::error::Error + std::marker::Send + 'static) -> std::option::Option<std::sync::Arc<error_chain::Backtrace>>`                                                                                              

error: aborting due to previous error

For more information about this error, try `rustc --explain E0046`.
error: Could not compile `dotenv`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

This happens on both rustc version 1.27.0 and the current nightly. If this is a hard dependency of the library it probably shouldn't be a feature.

**Comment by daboross** [comment]
I believe this should be fixed in 0.13.0 (released May 30th)? I can cornfirm when on a computer.

We should probably be testing a build with `--no-default-features` in Travis in either case, though.

**Comment by sstelfox** [comment]
I guess I should've checked for a new version first! That did the trick. I threw up a quick PR to add the test build with --no-default-features.
